os/net/blemgr/bledev_mgr_server.c

```diff
@@ -0,0 +1,82 @@
+/****************************************************************************
+ *
+ * Copyright 2021 Samsung Electronics All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied. See the License for the specific
+ * language governing permissions and limitations under the License.
+ *
+ ****************************************************************************/
+
+#include <tinyara/config.h>
+#include <stdio.h>
+#include <stdint.h>
+#include <tinyara/net/if/ble.h>
+#include <tinyara/ble/ble_manager.h>
+#include "bledev_mgr_server.h"
+
+#define BLE_DRV_TAG "[BLEDRV_SERVER]"
+
+static void ble_server_connected_cb(trble_conn_handle con_handle, trble_server_connection_type_e conn_type, uint8_t mac[TRBLE_BD_ADDR_MAX_LEN])
+{
+	return;
+}
+
+static void utc_cb_charact_a_1(trble_attr_cb_type_e type, trble_conn_handle linkindex, trble_attr_handle handle, void *arg)
+{
+	return;
+}
+
+static void utc_cb_desc_b_1(trble_attr_cb_type_e type, trble_conn_handle linkindex, trble_attr_handle handle, void *arg)
+{
+	return;
+}
+
+static trble_gatt_t gatt_profile[] = {
+	{
+		.type = TRBLE_GATT_SERVICE,
+		.uuid = {0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x01, 0x01, 0x01, 0x01, 0x01},
+		.uuid_length = 16,
+		.attr_handle = 0x006a,
+	},
+
+	{
+		.type = TRBLE_GATT_CHARACT, 
+		.uuid = {0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x01, 0x01, 0x01, 0x01, 0x02}, 
+		.uuid_length = 16, .property = TRBLE_ATTR_PROP_RWN | TRBLE_ATTR_PROP_WRITE_NO_RSP, 
+		.permission = TRBLE_ATTR_PERM_R_PERMIT | TRBLE_ATTR_PERM_W_PERMIT, 
+		.attr_handle = 0x006b, 
+		.cb = utc_cb_charact_a_1, 
+		.arg = NULL
+	},
+
+	{
+		.type = TRBLE_GATT_DESC, 
+		.uuid = {0x02, 0x29}, 
+		.uuid_length = 2, 
+		.permission = TRBLE_ATTR_PERM_R_PERMIT | TRBLE_ATTR_PERM_W_PERMIT, 
+		.attr_handle = 0x006c, 
+		.cb = utc_cb_desc_b_1, 
+		.arg = NULL,
+	},
+};
+
+// This config is for empty server.
+static trble_server_init_config g_server_null_config = {
+	ble_server_connected_cb,
+	true,
+	gatt_profile, sizeof(gatt_profile) / sizeof(trble_gatt_t)
```

Comment:
How about making new line for `sizeof(gatt_profile) / sizeof(trble_gatt_t)`?

Reply:
Ok, I fixed it